canonical/assets.ubuntu.com
# Getting the staging instance live to Prod

## Done

- Staging assets service with a new interface, and a separate datastore than production.

## QA

- Check out this feature branch
- Run the site using the command `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8017
- See the demo here https://manager.assets.staging.ubuntu.com/manager
